p2p/constants.py

```diff
@@ -69,3 +69,10 @@
 
 # Maximum allowed depth for chain reorgs.
 MAX_REORG_DEPTH = 24
+
+# Seal check frequency of headers downloaded during light/fast sync. Apparently 100 was the
+# optimal value determined by geth devs
+# (https://github.com/ethereum/go-ethereum/pull/1889#issue-47241762), but in order to err on the
+# side of caution, we use 192/4 to ensure we check 4 headers out of every batch (of length
+# MAX_HEADERS_FETCH).
+FAST_SYNC_SEAL_CHECK_FREQUENCY = 192 // 4
```

Comment:
Maybe it's clearer to specify the period here, like:

```py
# check one in 4 headers
FAST_SYNC_SEAL_CHECK_PERIOD = 4
```

Then, we aren't assuming this value is used with 192 headers, either.

Reply:
AIUI, geth checks 1 in 100, and as I mentioned below we're doing 1 in 48. Why would we do 1 in 4 instead?

Reply:
Sorry, I see now that the 'frequency` name was throwing me off. So I think my only issue with this is that it should be renamed to period. Frequency would be 1/period, ~0.02.

Reply:
What about `seal_check_random_sample_rate`? 

```
            sample_size = len(all_indices) // seal_check_random_sample_rate
            headers_to_check_seal = set(random.sample(all_indices, sample_size))
```